lib/resources/r4/medication_request.yaml

```diff
@@ -16,6 +34,7 @@ fields:
   required: 'No'
   type: CodeableConcept
   description: Reason for the current status
+  action: get
```

Comment:
don't think get is a supported action for tables

Reply:
Should it be removed then, or should it be another action? Or do we need confirmation on it?

Reply:
well i'm pretty sure it doesn't do anything differently present or absent - you can confirm with some testing locally

Reply:
looks like these are being used to keep fields out of the create table